specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/definitions.json

```diff
@@ -2358,6 +2390,40 @@
         }
       }
     },
+    "BackendPool": {
```

Comment:
Can you please update or add examples for your APIs where these new properties are shown to users as illustrations?

Reply:
Hey, we mean to update/add examples like these: [https://github.com/Azure/azure-rest-api-specs/blob/eb81a6e52c56de632657758e496f1a58f20cdd26/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/examples/ApiManagementApplyNetworkConfigurationUpdates.json](https://github.com/Azure/azure-rest-api-specs/blob/eb81a6e52c56de632657758e496f1a58f20cdd26/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/examples/ApiManagementApplyNetworkConfigurationUpdates.json), not an example within the definition, but in examples folder.